README.md

```diff
@@ -171,12 +172,10 @@ or later, which officially supported C++11.
 3. HuC firmware is needed for AVC/HEVC/VP9 low power encoding bitrate control, including CBR, VBR, etc. As of now, HuC firmware support is disabled in Linux kernels by default. Please, refer to i915 kernel mode driver documentation to learn how to enable it. Mind that HuC firmware support presents in the following kernels for the specified platforms:
    * APL/KBL: starting from kernel 4.11
    * CFL: starting from kernel 4.15
+   * ICL: starting from kernel 5.2
+   
+4. TGL: preliminary support and may not be fully functional on 5.4.2, need config i915.force_probe=* and no Huc, Guc loaded. [drm-tip](https://cgit.freedesktop.org/drm-tip) should work with more support and bug fixes.
```

Comment:
Suggestions:
1. Give range of kernels where TGL support appears and note that it is yet preliminary. See my variant:
> TGL: preliminary support and may not be fully functional. It requires Linux kernel 5.4+ loaded with i915.force_probe=* parameter to be tried. [drm-tip](https://cgit.freedesktop.org/drm-tip) should work with more support and bug fixes.

2. Don't mix up with HUC and GUC here. Just drop the text.

Reply:
agree, will change